media-libs/kcolorpicker/kcolorpicker-0.1.6.ebuild

```diff
@@ -30,7 +30,7 @@ DEPEND="${RDEPEND}
 S="${WORKDIR}/${MY_P}"
 
 src_configure() {
-	local mycmakeargs+=(
+	local mycmakeargs=(
 		-DBUILD_TESTS=$(usex test)
 	)
```

Comment:
This fix is in a wrong commit though :(

Reply:
Shoot, I would have happily fixed that before the merge.  Sorry, I'll do better.  Thanks for merging!